ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master__nfd.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: build/Dockerfile
+  from: os
+  to: ci-artifacts
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.8"
+resources:
+  '*':
+    limits:
+      memory: 2Gi
+    requests:
+      cpu: 2000m
+      memory: 2Gi
+tests:
+- as: e2e-master
```

Comment:
could you name it `nfd-operator-e2e-master`, for consistency? (just a suggestion)